lib/perl/Genome/Model/Tools/DetectVariants2/SpeedseqSv.pm

```diff
@@ -0,0 +1,109 @@
+#SpeedseqSv.pm
+
+
+package Genome::Model::Tools::DetectVariants2::SpeedseqSv;
+use warnings;
+use strict;
+
+use Genome;
+use File::Basename qw(basename dirname);
+use File::Spec;
+use IPC::System::Simple;
+use Genome::File::Tsv;
+
+class Genome::Model::Tools::DetectVariants2::SpeedseqSv {
+    is => 'Genome::Model::Tools::DetectVariants2::Detector',
+};
+
+	my $temp_directory = Genome::Sys->create_temp_directory();
+	my $pkg = 'Genome::Model::Tools::Speedseq::Sv';
+
+# For the Parameters what I need to do is make two hashes.
+# The first one will have the name of the command in the SV class and the letter calling it.  
+# The second Hash will have the class string name and the value wil be either a boolean or a file location.
+
+sub _detect_variants {
+	my $self = shift;
+	
+	my @fullBam = ($self->aligned_reads_input,$self->control_aligned_reads_input);
+
+	my $aligned_bams = join(',',@fullBam);
+
+	my %library = $self->get_parameter_hash();
+
+	while (my ($key, $value) = each(%library)){
+		print "$key => $value\n";
```

Comment:
Is this for debugging, or part of the output?
If it's for debugging, it could be `$self->debug_message("$key => $value")`.
if it's part of the output, it might be good to sort the keys so that the output is consistent across runs.